maintainers/maintainer-list.nix

```diff
@@ -12572,5 +12572,11 @@
     github = "mbprtpmnr";
     githubId = 88109321;
   };
+  simarra = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.